andela/the_spinners-frontend
# #169817494 manager reject-approve request

#### What does this PR do?
- Implement manager approve-reject a request 
#### Description of Task to be completed?
- Once a manager is logged in he should be able to view all requests sent to him. Once he clicks on `VIEW DETAILS` button on a certain request, he gets redirected to a page containing the details of the trip where he can `approve` or `reject` it.
#### How should this be manually tested?
- Login on barefoot nomad as a manager
- Click `My Approvals` link on the sidebar
- You get redirected to the page containing all trip requests assigned to you along with their status (`pending`, `rejected`, `approved`)
- Click on `VIEW DETAILS` button on a certain request, you can see details of such a request along with the `APPROVE` and `REJECT` buttons where you can `approve` or `reject` a request.
#### What are the relevant pivotal tracker stories?
[#169817494](https://www.pivotaltracker.com/story/show/169817494)
#### Screenshots
- Before clicking `VIEW DETAILS` button
<img width="1440" alt="Screen Shot 2020-03-11 at 09 27 14" src="https://user-images.githubusercontent.com/58309608/76392796-f5d8af80-637a-11ea-82d7-9f4aedc891c7.png">

- After clicking `VIEW DETAILS` button and before clicking `APPROVE` or `REJECT` button
<img width="1440" alt="Screen Shot 2020-03-11 at 09 27 57" src="https://user-images.githubusercontent.com/58309608/76392826-038e3500-637b-11ea-9fbe-a98a17115cfd.png">

- After clicking `REJECT` button
<img width="1440" alt="Screen Shot 2020-03-11 at 09 29 14" src="https://user-images.githubusercontent.com/58309608/76392846-0f79f700-637b-11ea-92a7-a01786ac9df0.png">


- After clicking `APPROVE` button
<img width="1440" alt="Screen Shot 2020-03-11 at 09 29 44" src="https://user-images.githubusercontent.com/58309608/76392872-199bf580-637b-11ea-9d4c-40231fbed7fe.png">
